go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/mendersoftware/go-lib-micro
+
+go 1.14
+
+require (
+	github.com/ant0ine/go-json-rest v3.3.2+incompatible
+	github.com/mendersoftware/inventory v0.0.0-20200522065057-a25956b1ec7b
+	github.com/mendersoftware/mendertesting v0.0.0-20200519115418-fd4090fcab1a
```

Comment:
ok, upgrade of mendertesting is required, right?

Reply:
I suppose it doesn't hurt :)